src/help.txt

```diff
@@ -0,0 +1,10 @@
+usage: jotdown [option] <file> ...
+
+arguments:
+    file            one or more djot source file(s)
```

Comment:
implementation seems to only allow 0 or 1 input files. we also only have 1 output file, so not sure how multiple input files would work.

Reply:
Once again a remnant from drafting, sorry about that. It doesn't make sense to have multiple inputs and concat them, I quickly realized that too.

I'll fix this as well